week-2/L-array-methods-2/exercise3.js

```diff
@@ -7,7 +7,7 @@
 var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUK(country) {
-  return; // complete this statement
+  return ukNations.includes("England"); // complete this statement
```

Comment:
This does not produce the desire output. If you run `exercise3.js` everything returns true